biorxiv/corpora_comparison/biorxiv_pmc_word_frequency_counts.py

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Compare Pubmed Central Corpus with bioRxiv Corpus
+
+# In[1]:
+
+
+get_ipython().run_line_magic('load_ext', 'autoreload')
+get_ipython().run_line_magic('autoreload', '2')
+
+from collections import Counter, defaultdict
+import csv
+from pathlib import Path
+import pickle
+import string
+import sys
+
+sys.path.append("../../modules/")
```

Comment:
Sorry to make repo structure comments one PR after a refactor, but if you set up your repo as a python module then you can use standard import syntax for importing `document_helper` without having to tinker with the `PYTHONPATH`

Reply:
Noted! I'll defer this point for another pull request as this one has gotten a bit big on changes.